core/src/main/python/wlsdeploy/yaml/yaml_translator.py

```diff
@@ -102,7 +102,7 @@ class PythonToYaml(object):
     _class_name = 'PythonToYaml'
     # 4 spaces
     _indent_unit = '    '
-    _requires_quotes_chars_regex = '[:{}\[\],&*#?|<>=!%@`-]'
+    _requires_quotes_chars_regex = '[\r\n:{}\[\],&*#?|<>=!%@`-]'
```

Comment:
Both files need updates to copyright dates and current format:

Copyright (c) [year,] year, Oracle and/or its affiliates.

Reply:
Copyright (c) 2019, 2021, Oracle and/or its affiliates.
Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.